valhalla/midgard/sequence.h

```diff
@@ -335,6 +335,47 @@ template <class T> class sequence {
     return;
   }
 
+  // sort the file based on the predicate, and outputs to output_seq
```

Comment:
Would it make sense to remove the old `sort` function if it's now unused and untested?

Reply:
its not unused. we do use it elsewhere, but we could just replace it with this one imho. ive got some local changes on top of this that im playing aroudn with let me see about that

Reply:
done